src/saga/adaptors/torque/torquejob.py

```diff
@@ -116,27 +116,21 @@ def log_error_and_raise(message, exception, logger):
 def _torque_to_saga_jobstate(torquejs):
     """ translates a torque one-letter state to saga
     """
-    if torquejs == 'C': # Torque "Job is completed after having run."
-        return saga.job.DONE
-    elif torquejs == 'F': # PBS Pro "Job is finished."
-        return saga.job.DONE
-    elif torquejs == 'H': # PBS Pro and TORQUE "Job is held."
+    if torquejs == 'H': #  "Job is held."
         return saga.job.PENDING
-    elif torquejs == 'Q': # PBS Pro and TORQUE "Job is queued(, eligible to run or routed.)
+    elif torquejs == 'Q': # "Job is queued(, eligible to run or routed.)
         return saga.job.PENDING
-    elif torquejs == 'S': # PBS Pro and TORQUE "Job is suspended."
+    elif torquejs == 'S': # "Job is suspended."
         return saga.job.PENDING
-    elif torquejs == 'W': # PBS Pro and TORQUE "Job is waiting for its execution time to be reached."
+    elif torquejs == 'W': # "Job is waiting for its execution time to be reached."
         return saga.job.PENDING
-    elif torquejs == 'R': # PBS Pro and TORQUE "Job is running."
+    elif torquejs == 'R': # "Job is running."
         return saga.job.RUNNING
-    elif torquejs == 'E': # PBS Pro and TORQUE "Job is exiting after having run"
+    elif torquejs == 'E': # "Job is exiting after having run"
```

Comment:
In Slurm, I opened a PR to have the equivalent of `E` (`CG`) interpreted as `CANCELED`, because at this point it is inevitable that the job will finish, and it can stay in `CG` rather long   for some reason, at least on BW and Stampede, and thus the RP shutdown is delayed quite a bit.  This is semantically not fully correct though.  Either way, we should keep it consistent I guess, and either purge the slurm PR, or also use that interpretation here and in the other adaptors.  

Thoughts?